rootfs/etc/systemd/system/bluetooth-workaround.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description="Bluetooth workaround"
+
+[Service]
+Type=oneshot
+ExecStart=mkdir -p /var/lib/bluetooth
+Before=bluetooth.service
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
does this ensure that this will run before the bluetooth service is started?

Reply:
Yes. Exactly. The "Before" will wait for this to finish before starting bluetooth service. I even destroyed the directory on purpose and this will fix it.

Reply:
I am blind and didn't notice that line, thanks. :facepalm: